### ISSUE TITLE
Default theme: color palette

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5-design#114]
The [default theme](105) must integrate easily with web applications and pages, which means it must be unobtrusive but good–looking. Though changing colors with [CSS preprocessors](110) is quite easy, a default color palette must be chosen.

The one suggested in #105 is:

![image](https://cloud.githubusercontent.com/assets/1099479/12643121/78ab2cc0-c5ba-11e5-8aca-cfa5dfcc2161.png)

What would you like CKEditor 5 to look like?

**Note**: The palette must meet [AA contrast requirements](106) in terms of accessibility.

**Comment by Reinmar** [comment]
We also need to style focused controls, so this pallet must consider a special, distinguishable color for it. Not like here on GitHub where I can't see that the "Comment" button is focused ;/

**Comment by oleq** [comment]
@Reinmar I realised that the blue in this palette doesn't meet WCAG 2.0 AA so the color of buttons will change for sure.

> We also need to style focused controls

I [proposed](105) a blue shadow for active controls (i.e. input fields) just like most websites/desktop applications (simply focus address bar in Chrome)

![image](https://cloud.githubusercontent.com/assets/1099479/12611502/1fb22f74-c4ed-11e5-8510-70dbe1e254ad.png)

but I'm not quite sure it also must meet WCAG 2.0 AA (clearly doesn't) because it's a kind of secondary feedback: you either see a caret in an input or the button changes the color (to be decided) as a whole.

**Comment by Reinmar** [comment]
There's no doubt that this is enough for light colored fields, but we must also consider how this outline will be visible around the blue "Save" button.

Blue outline for focused fields makes sense. If this outline will be thick enough, then most likely it will be enough even for controls matching its color.

**Comment by oleq** [comment]
> There's no doubt that this is enough for light colored fields, but we must also consider how this outline will be visible around the blue "Save" button.

I expect an active "Save" button to have a different color and that blue outline. The colors need research, as I mentioned, because they don't meet contrast standards.

**Comment by koleary** [comment]
I would consider going to green for the action buttons. It offers additional semantic emphasis (green = go)  that this is the primary action.

**Comment by oleq** [comment]
> I would consider going to green for the action buttons. It offers additional semantic emphasis (green = go) that this is the primary action. It also disambiguates "action" (green) from "link" (almost universally understood as blue).

Good point!

**Comment by Bojhan** [comment]
This is still missing focus styles.

For it to be accepted in Drupal for example it must meet WCAG AA, but keep in mind about contrast that is ornamental and functional.As borders can often serve as enough indication, if ratios are correct.

**Comment by oleq** [comment]
@Bojhan What I proposed in this issue is more a test palette than something final. It will certainly be extended and tuned to meet various requirements. In fact, there aren't enough UI components in the project at the moment to test it properly, so stay tuned.
